src/ZenstruckMessengerTestBundle.php

```diff
@@ -65,7 +71,30 @@ public function process(ContainerBuilder $container): void
                 }
             }
 
-            $registry->addMethodCall('register', [$name, new Reference($id)]);
+            $transportRegistry->addMethodCall('register', [$name, new Reference($id)]);
         }
+
+        $busRegistry = $container->getDefinition('zenstruck_messenger_test.bus_registry');
+        foreach ($container->findTaggedServiceIds('messenger.bus') as $id => $tags) {
+            $name = $this->generateAliasName($id);
+            $busRegistry->addMethodCall('register', [$name, new Reference($name)]);
+            $container->register($name, TestBus::class)
+                ->setAutowired(true)
+                ->setPublic(true)
+                ->setDecoratedService($id)
+            ;
+        }
+    }
+
+    private function generateAliasName(string $id): string
```

Comment:
Shamefully stollen from stackoverflow but I need to look for a better implementation if there is and check if it fits all the required needs.

It was only for a quick poc. :innocent:

Reply:
Can the [Target](https://github.com/symfony/symfony/blob/767e0f9b7e2211ee707dfc19a83726671d9d3247/src/Symfony/Component/DependencyInjection/Attribute/Target.php#L26) object's constructor logic be used to determine?

Reply:
what about `$name = "$id.test-bus"` or something hyper simple like that?

Reply:
It looks like it works. (at least tests still runs)

I wanted that the alias names match the ones usually created by Symfony but maybe it's not relevant or useful at any point.